ansible/roles_ocp_workloads/ocp4_workload_gitops_bootstrap/defaults/main.yml

```diff
@@ -13,3 +13,9 @@ ocp4_workload_gitops_bootstrap_helm_values: []
 
 ocp4_workload_gitops_bootstrap_health_retries: 60
 ocp4_workload_gitops_bootstrap_health_ignore: true
+
+# wait 15 mninutes for apps to roll out
+ocp4_workload_gitops_bootstrap_application_wait: true
+ocp4_workload_gitops_bootstrap_application_health_retries: 90
+ocp4_workload_gitops_bootstrap_application_health_ignore: true
+ocp4_workload_gitops_bootstrap_application_label: "demo.redhat.com/catalogItem = true"
```

Comment:
This label will be our interface with the external world (GitOps repositories), so I want to be thoughtful about its name.

I believe naming it **demo.redhat.com/application** makes it less presumptive and more flexible.

Reply:
I also don't think "= true" is necessary.

Reply:
Finally, I see this is defined as a variable, but I wouldn’t encourage changing it in agV. The reason is the potential inconsistency across different CIs and GitOps repositories. This opens it up for changes without providing any clear benefit, as far as I know.

Reply:
> I also don't think "= true" is necessary.

I like that it's specific.  This allows us to set it to "false" temporarily, if it's failing but we want to allow it.

Reply:
> Finally, I see this is defined as a variable, but I wouldn’t encourage changing it in agV. The reason is the potential inconsistency across different CIs and GitOps repositories. This opens it up for changes without providing any clear benefit, as far as I know.

Removing the variable makes the contract stronger. 🦾

Reply:
It's a binary label now.  "demo.redhat.com/application"
It's no longer a variable.